tests/test_submit.py

```diff
@@ -100,6 +104,24 @@ def test_submit_with_config(self):
             assert sub_config_data[SUB_CLI_CONFIG_KEY_SUBMISSION_ID] == "mock_submission_id"
             assert sub_config_data[SUB_CLI_CONFIG_KEY_SUBMISSION_UPLOAD_URL] == "directory to use for upload"
 
+    def test_submit_when_already_completed(self):
+        test_submission_ws_client = SubmissionWSClient()
+
+        with patch.object(self.submitter, 'submission_ws_client', test_submission_ws_client), \
+                patch.object(test_submission_ws_client, 'get_submission_status', return_value='UPLOADED') as mock_get_submission_status, \
+                patch.object(self.submitter, 'submission_dir', self.test_sub_dir), \
```

Comment:
Why does this need to be patched, isn't it set in `setUp` when the submitter is constructed?